### PR TITLE
Tag for remote esasky tests

### PR DESCRIPTION
**Pull request by imbasimba**
I want to be able to distinguish real users from just tests in the ESASky servers. Suggestions on how to do this better are welcome.

### CODE REVIEW

**Comment by keflavich** [comment]
I like this approach.  :+1:

**Comment by bsipocz** [comment]
This looks good to me, too. Merging once travis gives green.

File: astroquery/esasky/core.py [hunk 1]
```diff
@@ -762,7 +764,8 @@ def _fetch_and_parse_json(self, object_name):
         url = self.URLbase + "/" + object_name
         response = self._request('GET', url,
                                  cache=False,
-                                 headers={'User-Agent': ('astropy:astroquery.esasky.{vers}'.format(vers=version.version))})
+                                 headers={'User-Agent':
```
Comment (bsipocz) [inline astroquery/esasky/core.py]: there is whitespace in this line

File: astroquery/esasky/core.py [hunk 2]
```diff
@@ -785,7 +788,8 @@ def _send_get_request(self, url_extension, request_payload, cache):
         url = self.URLbase + url_extension
         return self._request('GET', url, params=request_payload,
                              timeout=self.TIMEOUT, cache=cache,
-                             headers={'User-Agent': ('astropy:astroquery.esasky.{vers}'.format(vers=version.version))})
+                             headers={'User-Agent':
```
Comment (bsipocz) [inline astroquery/esasky/core.py]: whitespace again

File: astroquery/esasky/tests/test_esasky_remote.py [hunk 3]
```diff
@@ -6,22 +6,23 @@
 
 from ... import esasky
 
+ESASkyClass = esasky.core.ESASkyClass()
```
Comment (bsipocz) [inline astroquery/esasky/tests/test_esasky_remote.py]: add an extra line here

File: astroquery/esasky/tests/test_esasky_remote.py [hunk 4]
```diff
@@ -6,22 +6,23 @@
...
 
 @remote_data
 class TestESASky:
-
+    ESASkyClass.__isTest = "Remote Test"
     def test_esasky_query_region_maps(self):
```
Comment (bsipocz) [inline astroquery/esasky/tests/test_esasky_remote.py]: add an extra line before this one

**Pull request merged**